logistics.md

```diff
@@ -203,14 +261,349 @@ these steps:
 Open a terminal and type `echo $SHELL`. If it reads `/bin/bash` then you are all set! If it does not,
 then for all parts of the subsequent instructions, whenever the instructions read “open a terminal,”
 please assume you are to open a terminal, type bash, and the proceed with the instructions as specified.
+
+###### **Git**
+
+You most likely already have Git installed.
+To check if you do:
+- On Debian -  type `sudo apt-get install git` inside the terminal
+- On Fedora - type `sudo yum install git` inside the terminal.
+
+###### **VSCode**
+Go to https://code.visualstudio.com/ and click the download button, then run the `.exe` file.
```

Comment:
I think this is the windows section and  not the linux section, no?

Reply:
woops will fix!

Reply:
those are the instructions i found here: https://school.brainhackmtl.org/setup/